### DISCUSSION TITLE
Does openmvg & openmvs support the16bits .TIFF image?

### DISCUSSION THREAD

**Discussion by Duuuuu** [openMVG/openMVG#2244] [Q&A] [answered]
Well, I'm doing some work with the UAV multi-band image, which is 16bits .tiff format. I'm wondering if openmvg & openmvs support this format to do the sfm and 3d reconstruction piplines? If there have been some work about this, could you give me some guide or notes? If not, then, will you handle this issue and add this feature in someday or oneday?

I have done some search, just like this links: https://www.opendronemap.org/2020/02/odm-0-9-8-adds-multispectral-16bit-tiffs-support-and-moar/ . Maybe this can give you some help!

Thanks so much!

**Comment by pmoulon** [comment, marked as answer]
We don't support it as 16bits. But I do think that if you compress to 8 bits you would still have better results than if you have used directly 8 bits image. Since the compression will still keep some of the details by compressing the dynamic of the image.

**Reply by Duuuuu** [reply]
The compression will results in loss of spectral information, which is important in photogemoetry and remote sensing field. Maybe I should consider another solution to do this. But, also, thanks very much for your answer!  (*^▽^*)
